core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -421,7 +421,7 @@ trait TestDriverComponent extends DriverComponent with DataAccess with DefaultIn
       "ns",
       "testConfig1",
       Some("Sample"),
-      None,
+      Some(Map.empty[String, AttributeString]),
```

Comment:
It was never correct to have prerequisites set as None because the unmarshaller in MethodConfigurationComponent only ever unmarshals them as Some(Map.empty)

prerequisites are an unused/defunct concept, however